solutions/01_hello_html/age.js

```diff
@@ -0,0 +1,9 @@
+var age = readString("what is your age?");
+
+console.log(age);
+
+function submitPress() {
```

Comment:
sweet! does this work for you? do you need to `preventDefault` (look it up if you don't know what it means)?